validate.swift

```diff
@@ -147,8 +147,11 @@ func downloadJSONSync<Payload: Decodable>(url: String, timeout: Int = 10) -> Res
         
     case .success(let data):
         do {
+            let payloadString = String(describing: String(data: data, encoding: .utf8))
+            print("payloadString: \(payloadString)")
```

Comment:
So when there's something wrong with the API call, instead of `Payload` we get something along the lines of this:

```
{
  "message": "Bad credentials",
  "documentation_url": "https://developer.github.com/v3"
}
```

It feels like coping with this type of error message might be worthwhile, I'd imagine there are a few things that could go wrong here and that they'd all be returned in a similar way. If we could report this message instead of `The data couldn’t be read because it is missing` then that might be worth doing? What do you think?

Reply:
👍 I think that makes perfect sense to bubble up the payload string in case we can't parse it.

Reply:
Yah agreed, should be able to catch it in the `downloadSync` closure and return an appropriate message.